frontend/packages/console-app/src/components/user-preferences/language/const.ts

```diff
@@ -4,3 +4,5 @@ export const supportedLocales = {
   ko: '한국어',
   ja: '日本語',
 };
+
+export const PREFERRED_LANGUAGE_LOCAL_STORAGE_KEY = 'bridge/preferredLanguage';
```

Comment:
I would call this something like `last-language` or `cached-language` to make it more clear what it is. The user preference is the source of truth.